src/index.html

```diff
@@ -11,12 +11,117 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <!-- font -->
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <!-- css -->
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- header -->
+    <header class="header">
+      <!-- logo -->
+      <a
+        href="#"
+        class="logo"
```

Comment:
Add font-size: 0 to `logo `class to remove this extra space
![image](https://github.com/mate-academy/layout_moyo-header/assets/80700247/25ce3a92-47df-4c6c-a64d-8fc7731c6764)